cost/aws/reserved_instances/report_by_bc/README.md

```diff
@@ -21,6 +21,12 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer